lib/Utils/Logging.pm

```diff
@@ -183,8 +184,7 @@ This method will call several other log gathering methods from this class.
 =cut
 
 sub export_logs {
-    select_log_console();
-    save_screenshot();
```

Comment:
Someone put in these `save_screenshot` calls in on purpose. I am not sure it's a good idea to remove those. I suggest you show in more details in the git commit message how the output looked in before and how it's better now (if it actually is)

Reply:
As I switched from log console to serial terminal, do we still really need this?

Reply:
Well, I don't know that so you need to convince us :)

Reply:
This does not make sense. After my change we don't switch to log console so we do not have to take picture of it.

What I'm saying is that before we switched to log console, took screenshot of it and then do something with it. But now, we don't switch to it at all.

Reply:
oh, yes. I see that now. Forgive me, I read it the other way around where a screenshot was taken from wherever we would come frome *before* we select the log console. So all is good.